src/Momento.Sdk.Incubating/Responses/CacheDictionaryIncrementResponse.cs

```diff
@@ -1,13 +1,41 @@
 ﻿using Momento.Protos.CacheClient;
+using Momento.Sdk.Exceptions;
 
 namespace Momento.Sdk.Incubating.Responses;
 
-public class CacheDictionaryIncrementResponse
+public abstract class CacheDictionaryIncrementResponse
 {
-    public long? Value { get; private set; }
+    public class Success : CacheDictionaryIncrementResponse
+    {
+        public long? Value { get; private set; }
+        public Success(_DictionaryIncrementResponse response)
+        {
+            Value = response.Value;
```

Comment:
Can the response value be `null`? If not, please make the `Value` property non-nullable.